tests/Unit/Models/LegiScan/StateTest.php

```diff
@@ -13,7 +13,7 @@ public function statesHaveManyBills()
     {
         $state = State::factory()->create();
         Bill::factory()->create([
-            'state' => $state,
+            'state_id'    => $state->id,
```

Comment:
I assume we won't need to use `state_id` once the bill is updated to our own models, like the state was?